### DISCUSSION TITLE
Hetzner, ARM

### DISCUSSION THREAD

**Discussion by clickersmudge** [kubermatic/kubeone#2773] [Q&A]
Hi,
12. April 2023 Hetzner has added a new type of ARM-based machine to its offer.
[https://www.hetzner.com/press-release/arm64-cloud/](https://www.hetzner.com/press-release/arm64-cloud/)

Unfortunately, it cannot be created as it is available in conroller version 1.15.0.
Do you plan to support/update the new controller?
[https://github.com/hetznercloud/hcloud-cloud-controller-manager/releases/tag/v1.15.0](https://github.com/hetznercloud/hcloud-cloud-controller-manager/releases/tag/v1.15.0)

**Comment by kron4eg** [comment]
Here's the PR to test: https://github.com/kubermatic/kubeone/pull/2774

**Reply by kron4eg** [reply]
OK, so we merged the PR, now you can build kubeone from main branch and testdrive a bit.

**Reply by clickersmudge** [reply]
Hi @kron4eg.
Are you planning soon to release a new version of KubeOne?

**Reply by xmudrii** [reply]
@clickersmudge This update is going to be included in KubeOne 1.7 which is planned for mid/end of June. Until then, you can build KubeOne from the main branch if you want to test this.

**Reply by clickersmudge** [reply]
@kron4eg @xmudrii Hi, We have noticed that KKP version v2.23.0 has been released. This is great news for us. We would like to ask you if you are also planning to release a new version of KubeOne in the near future? This information will allow us to plan the update process.

**Reply by xmudrii** [reply]
@clickersmudge Yes, but we don't have a concrete ETA at this time. We hope that we'll be able to release KubeOne 1.7.0 in the next several weeks.

**Comment by clickersmudge** [comment]
Hello @kron4eg @xmudrii ,
We are just testing the new version of KubeOne 1.7 and KKP.
We were looking forward to supporting the Hetzner ARM machines.

```
cluster_autoscaler_max_replicas = 5
cluster_autoscaler_min_replicas = 1
cluster_name = "kubeone"
control_plane_type = "cax11"
control_plane_vm_count = 1
datacenter = "nbg1"
image = "103908130" // Image ID of Ubuntu ARM on Hetzner, https://docs.hetzner.cloud/#images-get-all-images
initial_machinedeployment_replicas = 1
ip_range = 	"10.0.0.0/16"
lb_type = "lb11"
network_zone = "eu-central"
os = "ubuntu"
ssh_public_key_file = "~/.ssh/id_rsa.pub"
worker_type = "cax11"
```
Does ARM support work on Control Plane and KKP installation or only on edge servers. We have noticed that when the Ubuntu 22.04 ARM image is selected as the image file, the machine-controller containers do not get up. From the machine-controller documentation it appears that https://hub.docker.com/r/kubermatic/machine-controller/tags?page=1 are built on linux/amd64. Will Kube One and KKP install on ARM type machines?

**Reply by xmudrii** [reply]
I created #2922 to verify and track this issue.
